etc/config-template.toml

```diff
@@ -137,6 +137,15 @@
 [pd]
 ## PD endpoints.
 # endpoints = []
+# The interval at which to retry a PD connection initialization. Set to 0 to disable retrying 
+# connection initialization.
+# retry-interval = 300
+# If the client observes the same error message on retry, it can repeat the message only
+# every `n` times. Set to 1 to disable this feature.
+# retry-log-every = 10
+# The maximum number of times to retry a PD connection initialization. Default is unset,
+# which is infinity.
+# retry-max-count =
```

Comment:
is it better to use 0 for retrying infinitely?

Reply:
I don't know. 🤷

Reply:
PTAL @siddontang

Reply:
Great question, because there may be a use case for zero meaning no retry. In MySQL, the value `-1` means "autosize". We could use it here to mean "not applicable"/unlimited.

Reply:
maybe if the value is less than 0, we can convert to Max Int64 in use. Or we can use a larger value like 10000 (maybe 1hour) by default. :-)

Reply:
I think default should be u64::MAX.

Reply:
In order to avoid needing to do some new config manipulation code (converting between isize and usize when reading and storing the configuration) I'll use `isize::MAX`.

Reply:
PTAL